vm/src/ser_de.rs

```diff
@@ -0,0 +1,199 @@
+use std::fmt;
```

Comment:
I'm not sure about the filename `ser_de.rs`. I would suggest `serialization.rs`. Other ideas are welcome!

Reply:
What about `py_serde.rs`?